binr/rahash2/rahash2.c

```diff
@@ -336,14 +407,18 @@ int main(int argc, char **argv) {
 		default: return do_help(0);
 		}
 	}
+	if (encrypt && decrypt) {
+		eprintf ("rahash2: Option -E and -D are incompatible with each other.\n");
+		return 1;
+	}
 	if (compareStr) {
 		int compareBin_len;
 		if (bsize && !incremental) {
 			eprintf ("rahash2: Option -c incompatible with -b and -B options.\n");
 			return 1;
 		}
-		if (b64mode) {
-			eprintf ("rahash2: Option -c incompatible with -d or -D options.\n");
+		if ((decrypt && !strcmp (decrypt, "b64")) || (encrypt && !strcmp (encrypt, "b64"))) {
```

Comment:
@radare thanks :smile:. But I didn't understand "put b64 in a plugin as 'crypto'". Can you explain somewhat more?